adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory.Platform/Microsoft.IdentityModel.Clients.ActiveDirectory.Platform.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <TargetFrameworks>net45;netstandard1.1;netstandard1.3;MonoAndroid8.1;Xamarin.iOS10</TargetFrameworks>        
+    <TargetFrameworks>net45;netstandard1.1;netstandard1.3;win81;uap10.0;MonoAndroid8.1;Xamarin.iOS10</TargetFrameworks>        
```

Comment:
This is a pretty big change and should be reflected in the commit message (maybe in a separate commit?). I wasn't able to get this to build locally, but I'll try some more. 

Also, you should update the adal packaging projects (adal/build/LayoutNugetPackage.proj and adal/build/*.nuspec) to include these 2 TFMs in the nuget package.

Reply:
@bgavrilMS , I'm leaving the packaging work to you. The packages doesn't work for the dev branch already. I have a few more things that needs to be added before it builds (though the uap10/win10 build needs a lot more before it can be validated + some additional work, which I consider out of scope for the PR)